src/lib/Storage.js

```diff
@@ -1,68 +1,110 @@
 import {AsyncStorage} from 'react-native';
 import CryptoJS from "react-native-crypto-js";
 import * as SecureStore from 'expo-secure-store';
-import moment from 'moment';
 
+// Provides read/write utilities to the device's persistend storage.
+export default class Storage {
+  static get ENCRPYTION_PREFIX() { return "encrypted"; }
+
+  static modelName(name) { return `${Storage.ENCRPYTION_PREFIX}_${name}`; }
```

Comment:
Nit-pick: Since we want to distinguish between encrypted and possibly unencrypted stuff (in the future), maybe rename it to `encryptedModelName`?